ruby_programming/git/lesson_a_deeper_look_at_git.md

```diff
@@ -30,13 +30,23 @@ Let's look at some ways we can change recent and distant history to fit our need
 - Squash commits together
 - Split up commits 
 
+#### Getting Set Up
+
+Before we get started with the lesson, let's create a git playground in which we can safely follow along with the code and perform history changing operations. Go to GitHub, and as you have in the past create a new repository. Call it whatever you'd like, and clone this repository on your local system. Now, let's `cd` into the repository we just cloned, and create some new files! Once you're in the repository follow along with the following commands. Look them up if you're confused about anything that's happening - the more practice we get using our Linux Terminal the better. 
```

Comment:
Couple small grammar things. Also I would drop the last bit of the last sentence, with where this is in the curriculum people should have plenty of experience using the CLI already.

```suggestion
Before we get started with the lesson, let's create a Git playground in which we can safely follow along with the code and perform history changing operations. Go to GitHub, and as you have in the past create a new repository. Call it whatever you'd like, and clone this repository to your local system. Now, let's `cd` into the repository we just cloned, and create some new files! Once you're in the repository follow along with the following commands. Look them up if you're confused about anything that's happening.
```